README.md

```diff
@@ -0,0 +1,17 @@
+# Stick-Hero
+An arcade game Sitick Man for Sugar.
```

Comment:
`An arcade game for Sugar ` should be a nice subtitle :)

Reply:
ok sure :+1: